deps.edn

```diff
@@ -1,5 +1,5 @@
 {:paths ["src" "resources"]
- :deps {org.clojure/clojure {:mvn/version "1.10.1"}}
+ :deps {org.clojure/clojure {:mvn/version "1.11.0-alpha4"}}
```

Comment:
I'm using update-k to return namespaced keywords for the coeffect map, but we can just use reduce-kv instead if you want

Reply:
I would prefer `reduce-kv` for now then. I see it in the code now, it makes sense. We can revisit this later since I feel like that function will be refactored anyways.

Reply:
sure, done. And you are right, it's pretty bad, I just didn't want to clean to much before we don't know where we are going :)